tests/phpunit/api/v3/ContributionTest.php

```diff
@@ -2554,11 +2557,11 @@ public function testRepeattransactionRenewMembershipOldMembership(): void {
 
     // So it seems renewing this expired membership results in it's new status being current and it being pushed to a future date
     $this->callAPISuccess('Contribution', 'repeattransaction', ['original_contribution_id' => $entities[0]['id'], 'contribution_status_id' => 'Completed']);
-    $membership = $this->callAPISuccessGetSingle('Membership', ['id' => $membership['id']]);
+    $membership = $this->callAPISuccessGetSingle('Membership', ['id' => $membership['id'], 'version' => 4, 'return' => ['end_date', 'status_id.name']]);
     // If this date calculation winds up being flakey the spirit of the test would be maintained by just checking
     // date is greater than today.
     $this->assertEquals(date('Y-m-d', strtotime('+ 1 month -1 day')), $membership['end_date']);
-    $this->assertEquals($newStatusID, $membership['membership_type_id']);
+    $this->assertEquals('Current', $membership['status_id.name']);
```

Comment:
@eileenmcnaughton why are we saying it is current when shouldn't it be New based on the function comment?

Reply:
@seamuslee001 well cos that is what it actually is - this doesn't change anything about the value - it just makes the check correct

Reply:
@eileenmcnaughton I think the function comment should be updated then otherwise I feel like future devs are still going to be confused

Reply:
I removed the bit that seems to be wrong - I think the other comments are right